apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/index.php

```diff
@@ -256,6 +281,65 @@ function wpcom_global_styles_in_use() {
 	return $global_styles_in_use;
 }
 
+/**
+ * Checks whether the site is exempt from Premium Global Styles because
+ * it was created before the Premium Global Styles launch date (2022-12-15)
+ * and had already customized its Global Styles.
+ *
+ * @param  int $blog_id Blog ID.
+ * @return bool Whether the site is exempt from Premium Global Styles.
+ */
+function wpcom_premium_global_styles_is_site_exempt( $blog_id = 0 ) {
+	// Sites created after we made GS a paid feature (2022-12-15) are never exempt.
+	if ( $blog_id >= 213403000 ) {
+		return false;
+	}
+
+	// If the exemption check has already been performed, just return if the site is exempt.
+	if ( wpcom_global_styles_has_blog_sticker( 'wpcom-premium-global-styles-exemption-checked', $blog_id ) ) {
+		return wpcom_global_styles_has_blog_sticker( 'wpcom-premium-global-styles-exempt', $blog_id );
+	}
+
+	// Non-Simple sites are edge cases; other conditions will determine whether they can use GS.
+	if ( ! defined( 'IS_WPCOM' ) || ! IS_WPCOM ) {
+		return false;
```

Comment:
I believe this should be `true`. With `false`, it would mean that a Business site that has used GS on their site, it would lose that feature when their subscription expires.